MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+
+The `.NET MAUI Button` control supports to customize the border color, image width, corner radius, background color, and more. The button control can be customized using the following properties:
+
+## Text Customization
+
+The text inside the button can be customized by its text color, font size, font attributes, font family and text alignment.
+
+### TextColor
+
+The `TextColor` property is used to customize the color of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" TextColor = "White">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.TextColor = Color.White;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with text color](images/Button_textcolor.png)
+
+### FontSize
+
+The `FontSize` property is used to customize the size of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" FontSize = "18">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontSize = 18;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with font size](images/Button_fontsize.png)
+
+### FontAttributes
+
+The `FontAttributes` property is used to customize the font style of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" FontAttributes = "Italic">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontAttributes = FontAttributes.Italic;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with fontattributes](images/Button_fontattributes.png)
+
+### FontFamily
+
+The `FontFamily` property is used to customize the font family of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" FontFamily = "Samantha-Demo">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontFamily = "Samantha-Demo";
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with fontfamily](images/Button_fontfamily.jpg)
+
+### TextAlignment
+
+The `HorizontalTextAlignment ` and `VerticalTextAlignment ` properties are used to customize the alignment of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" HorizontalTextAlignment="Center" VerticalTextAlignment="Center">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.HorizontalTextAlignment = TextAlignment.Center;
+button.VerticalTextAlignment = TextAlignment.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Background Customization
+
+The background of the button can be customized by its background color, border color, border width and corner radius.
```

Comment:
The background of the button can be customized by its background color, border color, border width, and corner radius.